### DISCUSSION TITLE
How do I create my own personalized profile page？

### DISCUSSION THREAD

**Discussion by cwp8** [Wrapping-2000/Wrapping-2000#1] [Q&A] [answered]
How do I create my own personalized profile page？

**Comment by cwp0** [comment, marked as answer]
Just click Use this template, create your own warehouse according to your own github name, and then modify the relevant parameters in the Readme.md.tpl file. As shown below：
<img width="914" alt="截屏2023-08-24 15 19 50" src="https://github.com/cwp0/cwp0/assets/110087455/1d2089a9-0d1c-4186-9048-6a1ab57097a8">
<img width="1084" alt="截屏2023-08-24 15 22 31" src="https://github.com/cwp0/cwp0/assets/110087455/30d5c93e-6996-43dc-a933-fee93a1e39e0">

**Reply by cwp8** [reply]
ok
